components/ai_chat/core/browser/conversation_driver.cc

```diff
@@ -190,7 +191,8 @@ ConversationDriver::GetVisibleConversationHistory() {
   return list;
 }
 
-void ConversationDriver::OnConversationActiveChanged(bool is_conversation_active) {
+void ConversationDriver::OnConversationActiveChanged(
+    bool is_conversation_active) {
```

Comment:
looks like there're a few unrelated formatting changes in this PR

Reply:
fixed